docs/src/release_notes/21.8.0.rst

```diff
@@ -0,0 +1,102 @@
+21.8.0
+======
+
+Direct runner on GKE
+--------------------
+
+Klio can now be run on the additional runner of DirectGKERunner.
+
+Under the hood, when using the new DirectGKERunner with a Klio job, Klio creates a GKE deployment. When GKE starts the containers for the deployment, it runs the Klio job in DirectRunner mode (just like you would locally with ``klio job run --direct-runner``.
+
+The difference between DirectGKERunner and DirectRunner is that the latter automatically acknowledges messages it reads from the Pub/Sub queue before running it through the defined pipeline. This logic has been adapted on DirectGKERunner to acknowledge the message once it’s considered done running through the pipeline (i.e. either it successfully completed the pipeline, was filtered out, or was dropped due to an error in processing). Without this ability, if a container gets OOM-killed or the job gets otherwise interrupted, then the failing or in-progress messages do not return to the Pub/Sub queue for re-delivery. Those messages would be lost.
+
+Using the DirectGKERunner for a Klio job comes with inherent limitations:
+
+* The Klio job must:
+    * be a streaming job (and therefore read event input from a Pub/Sub subscription or topic)
+    * not need any aggregation transforms (e.g. GroupByKey, CombinePerKey, Distinct, etc). Reshuffle & Flatten are also not supported.
+    * not process non-KlioMessages (klio-job.yaml::job_config.allow_non_klio_messages)
+    * be deployed manually (i.e. via ``klio job run [OPTIONS]``).
+
+* Observability
+
+Logs can be viewed via **kubectl** [#f0]_ commands.
+Klio now supports configuration fields to enable the use of **shumway,** [#f1]_ a library for sending metrics to an **FFWD** [#f2]_ agent.
```

Comment:
Small nits:
* Can you put the comma after `[#f1]_` instead of before? 
* "a FFWD agent" instead of "an FFWD agent"  since it's pronounced "fast-forward agent" and not "F.F.W.D"